default/main.py

```diff
@@ -51,8 +51,14 @@ async def run_queries(x_appengine_cron: Optional[str] = Header(None)):
                         ssl=ssl_context
                     ) as resp:
                     json = await resp.json()
+                    json['api_key'] = user.apikey
                     if resp.status != 200:
                         logging.error(resp.text())
                         raise HTTPException(400, 'Bad request')
-                    httpSession.post(user.webhook, data=json)
+                    async with httpSession.post(user.webhook, json=json, ssl=ssl_context) as post:
+                        post_response = await post.text()
+                        if post.status != 200:
+                            logging.error(post.status)
+                            logging.error(post_response)
```

Comment:
I think it should be one log msg.
Imagine when system is running and many users are making requests, then if one receives 403 and the other one 400 you can get logs this way: 403, 400, response, response and you can;t be easly sure which is which.